src/components/Points/CampaignPromo.tsx

```diff
@@ -54,78 +59,71 @@ export const CampaignPromo = () => {
+          <Box position="relative">
+            {promotedCampaign.iconUrl && (
+              <img
+                alt={'Campaign icon'}
+                src={promotedCampaign.iconUrl}
+                style={{ maxWidth: '176px', maxHeight: '80px' }}
+              />
+            )}
+          </Box>
+        ) : (
+          <Typography variant="h1" fontWeight={700} maxWidth="80%" className={css.gradientText} mb={2}>
+            {promotedCampaign.name}
+          </Typography>
+        )}
 
         <Typography mt={5} variant="overline" color="text.secondary">
           Prize pool
         </Typography>
-        <Typography mt={1} variant="h4" fontSize="27px" fontWeight={700}>
-          600,000 <span style={{ color: '#2470FF' }}>MORPHO</span>
-        </Typography>
+        {promotedCampaign.rewardValue ? (
+          <Typography mt={1} variant="h4" fontSize="27px" fontWeight={700}>
+            {formatAmount(Number(promotedCampaign.rewardValue), 0)}{' '}
+            <span style={{ color: '#2470FF' }}>{promotedCampaign.rewardText}</span>
+          </Typography>
+        ) : (
+          <Typography mt={1} variant="h4" fontSize="27px" fontWeight={700}>
+            {promotedCampaign.rewardText}
+          </Typography>
+        )}
         <Typography variant="body2">& Safe points</Typography>
         <Typography variant="body2" color="text.secondary" mt={5}>
-          <ExternalLink href="https://dub.sh/morphocampaign" isPartner>
-            Learn more
-          </ExternalLink>{' '}
-          and participate in campaign:
+          Participate in campaign:
         </Typography>
         <Stack direction="row" alignItems="center" spacing={2} mt={2}>
-          <ExternalLink variant="button" color="primary" href={safeAppUrl} target="_blank" isPartner>
-            Safe App
-          </ExternalLink>
-          <Typography>or</Typography>
-          <ExternalLink
-            isPartner
-            href="https://app.morpho.org/"
-            target="_blank"
-            variant="button"
-            sx={{
-              color: '#2470FF',
-              '&:hover': {
-                backgroundColor: ({ palette }) => `rgba(36, 112, 255, ${palette.action.hoverOpacity})`,
-              },
-            }}
-          >
-            Morpho
-          </ExternalLink>
+          {safeAppUrl && (
+            <>
+              <ExternalLink variant="button" color="primary" href={safeAppUrl} target="_blank" isPartner>
+                Safe App
+              </ExternalLink>
+              <Typography>or</Typography>
+            </>
+          )}
+
+          {promotedCampaign.partnerUrl && (
+            <ExternalLink
+              isPartner
+              href={promotedCampaign.partnerUrl}
+              target="_blank"
+              variant="button"
+              sx={{
+                color: '#2470FF',
+                '&:hover': {
+                  backgroundColor: ({ palette }) => `rgba(36, 112, 255, ${palette.action.hoverOpacity})`,
```

Comment:
This blue color still belongs to one specific campaign or is it the same for every campaign?

Reply:
Its the same on every campaign. There is no configurable color atm. So we could change it but only to a constant other color.
cc @TanyaEfremova

Reply:
I changed all colors and styles according to the new designs:
![Screenshot 2024-09-19 at 16 43 07](https://github.com/user-attachments/assets/5c2172e9-bcea-4d4f-80a9-8cca9181b540)